src/memray/reporters/templates/base.html

```diff
@@ -144,6 +144,7 @@ <h5 class="modal-title" id="helpModalLabel">How to interpret {{ kind }} reports<
     const merge_threads = {{ merge_threads|tojson }};
     const memory_records = {{ memory_records|tojson }};
     const inverted = {{ inverted|tojson }};
+    const temporal = packed_data["high_water_mark_by_snapshot"] != null;
```

Comment:
This isn't fully correct - `--temporal --leaks` flamegraphs don't have `high_water_mark_by_snapshot`. Let's instead check:
```suggestion
    const temporal = packed_data.intervals != null;
```

That condition is true both with and without `--leaks`